lntest/itest/lnd_remote_signer_test.go

```diff
@@ -0,0 +1,247 @@
+		// We always use the mainnet coin type for our BIP49/84
+		// addresses!
+		CoinType: 0,
+		Account:  0,
+		Xpub: "tpubDDWAWrSLRSFrG1KdqXMQQyTKYGSKLKaY7gxpvK7RdV3e3Dkhvu" +
+			"W2GgsFvsPN4RGmuoYtUgZ1LHZE8oftz7T4mzc1BxGt5rt8zJcVQi" +
+			"KTPPV",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyMultiSig),
+		Xpub: "tpubDDXFHr67Ro2tHKVWG2gNjjijKUH1Lyv5NKFYdJnuaLGVNBVwyV" +
+			"5AbykhR43iy8wYozEMbw2QfmAqZhb8gnuL5mm9sZh8YsR6FjGAbe" +
+			"w1xoT",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyRevocationBase),
+		Xpub: "tpubDDXFHr67Ro2tKkccDqNfDqZpd5wCs2n6XRV2Uh185DzCTbkDaE" +
+			"d9v7P837zZTYBNVfaRriuxgGVgxbGjDui4CKxyzBzwz4aAZxjn2P" +
+			"hNcQy",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyHtlcBase),
+		Xpub: "tpubDDXFHr67Ro2tNH4KH41i4oTsWfRjFigoH1Ee7urvHow51opH9x" +
+			"J7mu1qSPMPVtkVqQZ5tE4NTuFJPrbDqno7TQietyUDmPTwyVviJb" +
+			"GCwXk",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyPaymentBase),
+		Xpub: "tpubDDXFHr67Ro2tQj5Zvav2ALhkU6dRQAhEtNPnYJVBC8hs2U1A9e" +
+			"cqxRY3XTiJKBDD7e8tudhmTRs8aGWJAiAXJN5kXy3Hi6cmiwGWjX" +
+			"K5Cv5",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyDelayBase),
+		Xpub: "tpubDDXFHr67Ro2tSSR2LLBJtotxx2U45cuESLWKA72YT9td3SzVKH" +
+			"AptzDEx5chsUNZ4WRMY5h6HJxRSebjRatxQKX1uUsux1LvKS1wsf" +
+			"NJ2PH",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyRevocationRoot),
+		Xpub: "tpubDDXFHr67Ro2tTwzfWvNoMoPpZbxdMEfe1WhbXJxvXikGixPa4g" +
+			"gSRZeGx6T5yxVHTVT3rjVh35Veqsowj7emX8SZfXKDKDKcLduXCe" +
+			"WPUU3",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyNodeKey),
+		Xpub: "tpubDDXFHr67Ro2tYEDS2EByRedfsUoEwBtrzVbS1qdPrX6sAkUYGL" +
+			"rZWvMmQv8KZDZ4zd9r8WzM9bJ2nGp7XuNVC4w2EBtWg7i76gbrmu" +
+			"EWjQh",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyStaticBackup),
+		Xpub: "tpubDDXFHr67Ro2tYpwnFJEQaM8eAPM2UV5uY6gFgXeSzS5aC5T9Tf" +
+			"zXuawYKBbQMZJn8qHXLafY4tAutoda1aKP5h6Nbgy3swPbnhWbFj" +
+			"S5wnX",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyTowerSession),
+		Xpub: "tpubDDXFHr67Ro2tddKpAjUegXqt7EGxRXnHkeLbUkfuFMGbLJYgRp" +
+			"G4ew5pMmGg2nmcGmHFQ29w3juNhd8N5ZZ8HwJdymC4f5ukQLJ4yg" +
+			"9rEr3",
+	}, {
+		Purpose:  keychain.BIP0043Purpose,
+		CoinType: harnessNetParams.HDCoinType,
+		Account:  uint32(keychain.KeyFamilyTowerID),
+		Xpub: "tpubDDXFHr67Ro2tgE89V8ZdgMytC2Jq1iT9ttGhdzR1X7haQJNBmX" +
```

Comment:
Seems we need to extend the existing RPCs to also have a way to obtain these xpubs. 

Tagging this here based on our offline convo

Reply:
Could extend `lncli wallet accounts list` to include this information